exporter/postgres/queries_containers.yml

```diff
@@ -0,0 +1,156 @@
+###
+#
+# Begin File: pod_metrics.yml
+#
+###
+ccp_container_network:
+  query: "SELECT interface,tx_bytes,tx_packets, rx_bytes,rx_packets from public.proc_network_stats()"
```

Comment:
Didn't we discuss putting the nodemx extension itself into the `monitor` schema that pgmonitor is already putting existing monitoring related objects into.

Also, discussing a bit more with yogesh, I know we'd originally discussed having the "container" in the name of these metrics, but it may be better to go back to just using "nodemx" in the name. Not all of the metrics may be just for containers in the future with this extension, so it may be better to use a more generic name for the metrics that come with it. Sorry to keep going back and forth on the naming with this, but we want to be sure and get it right since it can be very hard to change a metric name later since it's used all the way up the chain through to Grafana.

@jconway Any thoughts on schema location or metric naming here?

Reply:
updated exporter queries to use monitor schema. 
metric names changed from ccp_container_*  to ccp_nodemx_*

Note: pgnodemx extension has to be installed in monitor schema.